skorch/tests/test_helper.py

```diff
@@ -254,6 +254,209 @@ def test_equals_different_keys(self, sldict_cls):
         assert sldict0 != sldict1
 
 
+class TestSliceDataset:
+    @pytest.fixture(scope='class')
+    def data(self):
+        X, y = make_classification(100, 20, n_informative=10, random_state=0)
+        return X.astype(np.float32), y
+
+    @pytest.fixture
+    def X(self, data):
+        return data[0]
+
+    @pytest.fixture
+    def y(self, data):
+        return data[1]
+
+    @pytest.fixture
+    def custom_ds(self, data):
+        """Return a custom dataset instance"""
+        from skorch.dataset import Dataset
+        class MyDataset(Dataset):
+            """Simple pytorch dataset that returns 2 values"""
+            def __init__(self, X, y):
+                super().__init__(X, y)
```

Comment:
Why have the `__init__` implementation at all?

Reply:
True, I will remove this.